lib/owl/data/sequence.ex

```diff
@@ -0,0 +1,95 @@
+defmodule Owl.Data.Sequence.Helper do
+  @moduledoc false
+
+  defmacro defsequence_type(name, type) do
+    quote bind_quoted: [name: name, type: type], generated: true do
```

Comment:
`generated: true` is necessary here to prevent a "clause cannot match because of ..." warning due to `:inverse` and `:reverse` using the same escape sequence. See b69001f9a7d3689d58678d94f91835f00ce580c1 for the alternative.

Reply:
I think alternative solution with explicit option is better.

Reply:
Okay, cool! I'm going to force-push to remove this last commit, then.